Makefile

```
@@ -432,6 +433,14 @@ openapi/generate/rhsso: go-bindata openapi-generator
 	$(GOFMT) -w pkg/client/redhatsso/api
 .PHONY: openapi/generate/rhsso
 
+# fail if formatting is required
+code/check:
+	@if ! [ -z $$($(GOFMT) -l `find . -type f -name '*.go' -not -path "./vendor/*"`) ]; then \
```

I guess there is some value in remaining consistent with `code/fix` below, but generally speaking, this would better be rewritten as:
```suggestion
	@if ! [ -z "$$(find . -path './vendor' -prune -o -type f -name '*.go' -print0 | xargs -0 $(GOFMT) -l)" ]; then \
```
1. `""` prevents issues with spaces in expansion. `[ .. ]` does not do anything special with `$(...)`. The following, for example, will return a zero exit status: `[ -z $(echo foo -o -n bar) ]`, because the expansion string is tokenized due to the missing quotes (if you use `[[ ... ]]`, which is a bash builtin, bash does recognize that `$(...)` should be treated as a single argument to `-z`).
2. `find . -path './vendor' -prune -o ...` makes sure find does not traverse the `vendor` directory, if any (your version does post-filtering, which requires first finding all `.go` files within vendor as well)
3. `find ... -print0 | xargs -0` prevents any tokenization issues in case file paths contain spaces.